geonode/settings.py

```diff
@@ -268,18 +268,18 @@
 
 GEONODE_CONTRIB_APPS = (
     # GeoNode Contrib Apps
-    'geonode.contrib.dynamic',
-    'geonode.contrib.exif',
-    'geonode.contrib.favorite',
-    'geonode.contrib.geogig',
-    'geonode.contrib.geosites',
-    'geonode.contrib.nlp',
-    'geonode.contrib.slack',
-    'geonode.contrib.metadataxsl'
+    # 'geonode.contrib.dynamic',
```

Comment:
Enabling metadataxsl contrib module by default. Before all contribs are disabled by simply commenting this line

# GEONODE_APPS = GEONODE_APPS + GEONODE_CONTRIB_APPS

now if you want to enable a contrib module you must explictly uncomment the module following the same paradigm as the other settings.